custom_components/ha_washdata/manager.py

```
@@ -2200,8 +2216,36 @@ async def async_reload_config(self, config_entry: ConfigEntry) -> None:
 
         self._logger.info("Configuration reloaded successfully")
 
+    def _spawn_tracked(self, coro: Any) -> "Task[Any]":
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `coro`

(ANN401)